samples/cpp/squares.cpp

```diff
@@ -130,7 +130,7 @@ int main(int argc, char** argv)
     if( argc > 1)
     {
      names[0] =  argv[1];
-     names[1] =  "0";
+     names[1] =  0;
```

Comment:
Could you please remove `static` from `names` declaration too (doesn't make sense):

```.diff
-static const char* names
+const char* names
```

Reply:
Yes, of course. I fixed this and squashed it in a single commit.